hera_cal/vis_clean.py

```diff
@@ -0,0 +1,471 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 the HERA Project
+# Licensed under the MIT License
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+from collections import OrderedDict as odict
+from copy import deepcopy
+import argparse
+import datetime
+from six.moves import range, zip
```

Comment:
yeah but I'd like to keep these as a fail safe just in case we are adding code in the future and forget that we need to use updated py3k `range` and `zip`